app/javascript/article-form/elements/bodyPreview.jsx

```diff
@@ -1,6 +1,14 @@
 import { h } from 'preact';
 import PropTypes from 'prop-types';
 
+function getCoverImage(imageSrc) {
```

Comment:
Thank you for the contribution. I think it would be better if it was a pure functional react component rather than a "javascript function" because it is what it is, so:

```
const CoverImage = ({ className, imageSrc, imageAlt }) => (
  <div className={className}>
    <img src={imageSrc} alt={imageAlt} />
  </div>
);
```

In that case, we can also define the propTypes for the component.

```
CoverImage.propTypes = {
  ...
}
```

And we can use like this:

```
<CoverImage
  className="articleform__mainimage articleform__mainimagepreview"
  imageSrc={previewResponse.cover_image}
  imageAlt="cover"
/>
```

And code climate will like it also